apps/vaporgui/main.cpp

```diff
@@ -166,7 +166,6 @@ if (getenv("VAPOR_DEBUG"))
     fontFile = fontFile + "//arimo.ttf";
 
 	QFontDatabase fdb; 
-	int result = fdb.addApplicationFont(QString::fromStdString(fontFile));
```

Comment:
No

Reply:
good call! 

Fixed in commit 104b214 .